# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/shaderc/hotfix-no-3rd-party.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- shaderc-2024.0/CMakeLists.txt.vanilla	2024-04-22 19:16:25.822000000 +0200
+++ shaderc-2024.0/CMakeLists.txt	2024-04-22 19:16:32.969000000 +0200
@@ -120,7 +120,7 @@
 
 # Configure subdirectories.
 # We depend on these for later projects, so they should come first.
-add_subdirectory(third_party)
+#add_subdirectory(third_party)
 
 add_subdirectory(libshaderc_util)
 add_subdirectory(libshaderc)
--- shaderc-2025.3/CMakeLists.txt.vanilla	2025-07-09 13:01:12.328665437 +0200
+++ shaderc-2025.3/CMakeLists.txt	2025-07-09 13:02:02.132070935 +0200
@@ -140,9 +140,7 @@
 endif()
 
 add_custom_target(build-version
-  ${Python_EXECUTABLE}
-  ${CMAKE_CURRENT_SOURCE_DIR}/utils/update_build_version.py
-  ${shaderc_SOURCE_DIR} ${spirv-tools_SOURCE_DIR} ${glslang_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/build-version.inc
+  touch ${CMAKE_CURRENT_BINARY_DIR}/build-version.inc
   COMMENT "Update build-version.inc in the Shaderc build directory (if necessary).")
 
 function(define_pkg_config_file NAME LIBS)
